book_config.py

```diff
@@ -130,75 +111,43 @@ def __init__(self, db, ids, parent):
     @property
     def book(self):
         return self._book_settings[self._index]
-    
-    def edit_asin(self, val):
-        self.book.asin = val
 
-    def edit_shelfari_url(self, val):
-        http_string = 'http://www.'
+    def edit_goodreads_url(self, val):
+        goodreads_string = 'https://www.goodreads.com/'
         index = 0
-        if val[:len(http_string)] != http_string:
+        if val[:len(goodreads_string)] != goodreads_string:
```

Comment:
Minor: if we are keeping this then val.startswith(goodreads_string) would be clearer/more pythonic.